website/addons/wiki/tests/test_wiki.py

```diff
@@ -86,7 +86,7 @@ def test_wiki_url_with_path_get_returns_200(self):
         res = self.app.get(url, auth=self.user.auth)
         assert_equal(res.status_code, 200)
 
-    def test_wiki_url_with_edit_get_returns_404_with_no_write_permission(self):
+    def test_wiki_url_with_edit_get_returns_403_with_no_write_permission(self):
```

Comment:
What happens for a user that's not logged in?

I believe the endpoint should return a 401 (Unauthorized), but I don't see a test for that anywhere - or for that matter, a pre-existing test for that purpose.